webExtensionApis/study/schema.yaml

```diff
@@ -698,3 +698,13 @@
       - prefs: object of all created prefs and their names
 
     parameters: []
+
+  - name: getInternalTestingOverrides
```

Comment:
Not really relevant to this review, but why is there both a Yaml and JSON version of the schema? That seems like a duplication risk.

Reply:
Sure, created a tracking issue: https://github.com/mozilla/shield-studies-addon-utils/issues/264

Reply:
The JSON is generated from YAML.  We found it faster to iterate (less verbose, multi-line comments) in YAML.